shared/.pronto_spell.pws

```diff
@@ -347,16 +367,48 @@ optin
 optonline
 oscommerce
 otpauth
+pal
+pam
+pan
 param
 params
+pas
 pathname
 pavel
+pax
+paxl
+paxxl
+paxxxl
+pbl
+pbm
+pbn
+pbs
+pbx
+pbxl
+pbxxl
+pbxxxl
```

Comment:
Whitelist these in `.pronto-spell.yml` with a regexp instead to keep this list clean.
```yml
whitelist:
  - '^(?:[0-9a-fA-F]{3}){2}$' # hex colors, e.g. ffffff
  - '^p[abhlrtv]xxx?l$' # CSS class names, e.g. pvxxxl
  - '^m[bhltv]xxx?l$' # CSS class names, e.g. mbxxl
  - ^sub        # subclass, subquery, subnet
  - '[ai]ble$'  # joinable, deletable, invertible
  - '[eo]rs?$'  # accessor, shortener, approvers
  - ify$        # stringify, regexpify
  - ized?$      # tableize, parameterize, parameterized
  - ^XX         # front-end fancy component preloader text placeholder